app/templates/index.html

```diff
@@ -66,7 +66,7 @@ <h2>launch a gpu session</h2>
                 Password
             </label>
             <p id="create-password-help" class="form-text text-muted">
-                <strong>this is not your login password!</strong> this password will be used for your login to the <code>jupyter notebook</code> server and as authentication for killing your container
+                <strong>this is your gpu-box login password!</strong> this password will be used for your login to the <code>jupyter notebook</code> server and as authentication for killing your container
             </p>
             <input type="password" class="form-control" id="create-password" name="jupyter_pwd" placeholder="enter password">
```

Comment:
change this to match your modified call in `launchapp.py` line 61. 
```suggestion
            <input type="password" class="form-control" id="create-password" name="password" placeholder="enter password">
```